readthedocs/embed/views.py

```diff
@@ -68,29 +63,10 @@ class EmbedAPIBase(CDNCacheTagsMixin, APIView):
     permission_classes = [IsAuthorizedToViewVersion]
     renderer_classes = [JSONRenderer, BrowsableAPIRenderer]
 
-    @functools.lru_cache(maxsize=1)
-    def _get_project(self):
-        if self.unresolved_url:
-            project_slug = self.unresolved_url.project.slug
-        else:
-            project_slug = self.request.GET.get('project')
-        return get_object_or_404(Project, slug=project_slug)
-
-    @functools.lru_cache(maxsize=1)
-    def _get_version(self):
-        if self.unresolved_url:
-            version_slug = self.unresolved_url.version_slug
-        else:
-            version_slug = self.request.GET.get('version', 'latest')
-        project = self._get_project()
-        return get_object_or_404(project.versions.all(), slug=version_slug)
-
-    @cached_property
-    def unresolved_url(self):
-        url = self.request.GET.get('url')
-        if not url:
-            return None
-        return unresolve(url)
+    @property
+    def external(self):
+        # Always return False because APIv2 does not support external domains
```

Comment:
Is this documented somewhere? Seems like a random detail that would be confusing for users.

Reply:
Hrm, I think it's not explicitly documented.

* v2: https://docs.readthedocs.io/en/stable/api/v2.html#embed
* v3: https://docs.readthedocs.io/en/stable/api/v3.html#embed

We are not mentioned that external domains are allowed in v3 either --we only mention it in the response. I think this was on purpose because we originally didn't want to expose these external domains to people due to concerns about misuse, etc.

But yeah, we should probably re-think this, update the docs and communicate better how we are using external domains and how people can propose extra domains maybe.